components/02-molecules/accordion/accordion.twig

```diff
@@ -0,0 +1,53 @@
+{% set accordion__base_class = 'accordion' %}
+
+{% set expand_all__content %}
+  Expand All
+  {% include "@atoms/images/icons/_icon.twig" with {
+    icon__name: 'angle-down',
+    icon__blockname: accordion__base_class,
+  } %}
+{% endset %}
+
+{% set collapse_all__content %}
+  Collapse All
+  {% include "@atoms/images/icons/_icon.twig" with {
+    icon__name: 'angle-down',
+    icon__blockname: accordion__base_class,
+  } %}
+{% endset %}
+
+<div {{ bem(accordion__base_class) }} data-component-width="content">
+  <div {{ bem('inner', [], accordion__base_class) }}>
+    {% include "@atoms/typography/headings/heading.twig" with {
+      heading__level: '3',
+      heading__blockname: accordion__base_class,
+      heading: accordion__heading,
+    } %}
+    {% embed "@atoms/lists/list.twig" with {
+      list__base_class: 'controls',
+      list__blockname: accordion__base_class,
+      list__attributes: {'aria-label': 'Section controls'},
+    } %}
+      {% block list__content %}
+        {% include "@atoms/buttons/button.twig" with {
+          button__content: expand_all__content,
```

Comment:
- [ ] Can we wrap the buttons in `<li>`s? The accessibility checker in Storybook flagged this since buttons cannot be children of `<ul>`s. Even though it sounds great in VO and NVDA screen readers, we should add these in to pass tests/pass HTML validators. We can always make a bug-fix if this requires a re-work, not a blocker. 

<img width="965" alt="Screen Shot 2022-03-25 at 10 34 40 AM" src="https://user-images.githubusercontent.com/12991593/160141419-b699598e-09c9-4efe-ac88-7d562b269ae2.png">

Reply:
Done

Reply:
Perfection!